plopfile.js

```diff
@@ -1,6 +1,10 @@
+const pascalCase = require('pascal-case');
+
 module.exports = (plop) => {
   plop.setHelper('upperCase', (txt) => txt.toUpperCase());
 
+  plop.setHelper('inBraces', (txt) => `{${pascalCase.pascalCase(txt)}}`);
```

Comment:
This is necessary to get the braces around the component name without having spaces around the name (ex: `{SuperAwesomeComponent}` vs `{ SuperAwesomeComponent}` – the second version doesn't need anything extra but it would be out of sync with how we actually write this line).

Reply:
There's probably an easier way to do this but it's the only way I could think of.

Reply:
oh weird 🤔 could you leave that as a code comment?